addons/website_project/controllers/main.py

```diff
@@ -30,9 +30,19 @@ def extract_data(self, model, values):
         data = super().extract_data(model, values)
         if model.model == 'project.task' and values.get('email_from'):
             partners_list = request.env['mail.thread'].sudo()._mail_find_partner_from_emails([values['email_from']])
-            partner_id = partners_list[0].id if partners_list else False
-            data['record']['partner_id'] = partner_id
+            partner_id = partners_list[0] if partners_list else False
```

Comment:
We have to do that
```suggestion
            partner_id = partners_list[0] if partners_list else self.env['res.partner']
```

otherwise `partner_id.id` will crash since `False` has no attribute `id`...

I would also rename the variable

```suggestion
            partner = partners_list[0] if partners_list else self.env['res.partner']
```